packages/api/cms-api/src/user-permissions/entities/user-permission.entity.ts

```diff
@@ -0,0 +1,50 @@
+import { BaseEntity, Entity, PrimaryKey, Property } from "@mikro-orm/core";
+import { Field, ID, ObjectType, registerEnumType } from "@nestjs/graphql";
+/* eslint-disable @typescript-eslint/naming-convention */
+import { v4 } from "uuid";
+
+export enum UserPermissionSource {
+    MANUAL = "manual",
+    BY_RULE = "by rule",
```

Comment:
Enum keys and values should be the same, they might cause issues with GraphQL otherwise

Reply:
74e9bfc25a92377704e77d9f2e9e8e94da70b9e4